### ISSUE TITLE
As someone who is colorblind, i find it really hard to see the status of a page in not active sites

### ISSUE THREAD

**Issue by GertTimmerman** [statamic/cms#3173]
I am one of people who are colorblind. Sometimes it brings me in hillarious situations and people ask me: do you only see black and white :) But sometimes it brings me in situation that i can't function right because i don't see the right colors. 

For example, the status of pages in other sites in Statamic, see the picture below:

![statamic-status-multisite](https://user-images.githubusercontent.com/6702926/106266480-c1870f80-6228-11eb-82dd-164070effb2e.png)

For me, except from the EN, its super hard to see the status by the color in the dot before the site. I can't tell you which color is which, for me, they are all the same. I have to inspect the element to see which class is loaded.

So, is it possible to make more distinction between a page that don't exist in another site and a page that does exist in another site which is not active?

**Comment by arthurperton** [comment]
An outlined dot could be used for sites where the page doesn't exist:

<img width="326" alt="Screenshot 2021-02-11 at 13 57 14" src="https://user-images.githubusercontent.com/4067531/107639482-34ed4000-6c71-11eb-951d-5ef1be639dff.png">

Or even strikethrough:

<img width="325" alt="Screenshot 2021-02-11 at 13 55 02" src="https://user-images.githubusercontent.com/4067531/107639647-70880a00-6c71-11eb-8c0a-921bcecad0d4.png">

Is one of these solutions helpful? I would be glad to make a PR for the preferred option.

**Comment by jackmcdade** [comment]
Redesigned in v6.
